app/assets/stylesheets/application.scss

```diff
@@ -50,6 +50,11 @@ nav {
   color: green;
 }
 
+.flash_error {
+  text-align: center;
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.